documentation/modules/proc-downloading-product.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-product-downloads-{context}']
+= Downloading {ProductName}
+
+Download a ZIP file that contains the resources required for installation and examples for configuration.
+
+.Prerequisites
+
+* Access to {ReleaseDownload}.
+
+.Procedure
+
+. Download the `strimzi-xyz.zip` file from {ReleaseDownload}.
```

Comment:
it's `strimzi-x.y.z.zip`

Reply:
But the documentation is versioned, so we should use attributes to uses the actual numbers, right?